features/eolearn/tests/test_doubly_logistic_approximation.py

```diff
@@ -0,0 +1,77 @@
+import unittest
+
+import numpy as np
+from eolearn.core import EOPatch, FeatureType
+from eolearn.features.doubly_logistic_approximation import DoublyLogisticApproximationTask
+from datetime import datetime
+
+
+class TestDoublyLogisticApproximation(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        dates = [datetime(2017, 1, 1), datetime(2017, 1, 11), datetime(2017, 2, 7), datetime(2017, 2, 17),
```

Comment:
Instead of using hand-crafted values, could you perhaps change this so that it uses the test EOPatch from `example_data/TestEOPatch`?

Reply:
![Inverted mask](https://user-images.githubusercontent.com/19612783/87913538-92b90100-ca6f-11ea-9fae-3ae9bf041d85.png)
I found out that IS_VALID mask in the test patch appears to be inverted. The pixels marked with True have a value of nan.

Reply:
Not sure what you mean by inverted. `IS_VALID` is 0 when the image is invalid (clouds or no data) and vice versa otherwise, i.e. in the image below, the cloudy parts are masked where `IS_VALID` is `FALSE`

![image](https://user-images.githubusercontent.com/37869250/87939051-c0b23b80-ca97-11ea-9f57-34442d7cbff2.png)

For which temporal index in the TestEOPatch do you see this? I'm also not sure what you are plotting above.

Reply:
![image](https://user-images.githubusercontent.com/19612783/88031033-52738480-cb3c-11ea-9333-ce3fc275b6d0.png)
I should've specified what colors mean. Here is another picture. On the left is displayed which pixels of the `(FeatureType.DATA` `'ndvi')` are equal to nan. White - "is nan", black -"not nan, data".
On the right is displayed the `IS_VALID` mask where White-True, Black-False.

The pixels which are marked with mask `IS_VALID` as True don't actually have any data and vice versa.

The offender is the patch in eo-learn/features/eolearn/tests/TestInputs/TestPatch.

Reply:
You are right, thanks for finding this. This must be some weird patch. I'll see what the damage is, meanwhile could you please use a different test EOPatch? It's in `eo-learn/example_data/TestEOPatch`. This one has a proper valid mask, but you have to calculate te ndvi